### DISCUSSION TITLE
Railway app deployment and env

### DISCUSSION THREAD

**Discussion by justynpride** [HiEventsDev/Hi.Events#110] [Q&A]
I'm new to using docker so forgive me if I've missed something obvious. 
I've successfully deployed the docker image for the Railway App from GitHub. 
I can access the system, but want to be able to set a number of the env variables such as stripe and mail settings. 
I cannot see any of these variables in the Railway App system variables options. 
Thanks in advance for any helpful pointers.

The list of variables I get to see in Railway is below:

![image](https://github.com/user-attachments/assets/c522f446-aa9b-4968-a3f5-012b405bc072)

**Comment by daveearley** [comment]
Hi @justynpride,

You'll need to add them yourself. Railway should give you the option to add new variables. They also have a `RAW Edit` mode which allows you to edit multiple at once, which makes it a lot easier.

**Reply by justynpride** [reply]
@daveearley perfect! Thank has solved my conundrum. I'll carry on with configuring and testing.
